src/applications/ivc-champva/10-10D/helpers/utilities.js

```diff
@@ -40,36 +40,61 @@ export const applicantListSchema = (requireds, propertyList) => {
   };
 };
 
-/**
- * Gets the appropriate form key needed in the signature box so we can
- * corroborate who is signing the form.
- *
- * @param {object} formData All data currently in the form
- * @returns
- */
-export function getNameKeyForSignature(formData) {
-  let nameKey;
-  if (formData.certifierRole === 'sponsor') {
-    nameKey = 'veteransFullName';
-  } else if (formData.certifierRole === 'applicant') {
-    nameKey = 'applicants[0].applicantName';
-  } else {
-    nameKey = 'certifierName';
-  }
-
-  return nameKey;
-}
-
 // Extracting this to a function so there aren't a thousand identical
 // ternaries we have to change later
 export function sponsorWording(formData, isPosessive = true, cap = true) {
-  let retVal = '';
-  if (formData?.certifierRole === 'sponsor') {
-    retVal = isPosessive ? 'your' : 'you';
-  } else {
-    retVal = isPosessive ? 'sponsor’s' : 'sponsor';
-  }
-
```

Comment:
We no longer capture if the current user is the sponsor, so this logic is useless.

Reply:
This is marked as outdated because I pushed after leaving the comment, but the information is still correct.